src/sentry/search/eap/spans.py

```diff
@@ -145,7 +157,26 @@ def _resolve_terms(self, terms: event_filter.ParsedTerms) -> TraceItemFilter | N
         for item in terms:
             if isinstance(item, event_search.SearchFilter):
                 resolved_column, context = self.resolve_column(item.key.name)
-                if item.operator in constants.OPERATOR_MAP:
+                raw_value = item.value.raw_value
+                if item.value.is_wildcard():
+                    if item.operator == "=":
+                        operator = ComparisonFilter.OP_LIKE
+                    elif item.operator == "!=":
+                        operator = ComparisonFilter.OP_NOT_LIKE
+                    else:
+                        raise InvalidSearchQuery(
+                            f"Cannot use a wildcard with a {item.operator} filter"
+                        )
+                    # Slashes have to be double escaped so they are
+                    # interpreted as a string literal.
+                    raw_value = (
+                        str(item.value.raw_value)
+                        .replace("\\", "\\\\")
+                        .replace("%", "\\%")
+                        .replace("_", "\\_")
+                        .replace("*", "%")
+                    )
```

Comment:
Does this work with characters that are already escaped? like `foo:*a\*b*`?

Reply:
hmm i don't think so, but I'm going to leave this for now (its identical to the QB behaviour) and revisit it later